resources/views/layouts/master.blade.php

```diff
@@ -8,7 +8,7 @@
 
         <title>Rogue</title>
 
-        <link rel="icon" type="image/png" href="http://twooter.biz/Gifs/tonguecat.png">
+        <link rel="apple-touch-icon-precomposed" href="/assets/images/apple-touch-icon-precomposed.png">
```

Comment:
You know I gotta ask that if this isn't TongueCat, then what is it?

Reply:
Tongue Cat lives on in the "empty" component!!

![Screen Shot 2019-11-14 at 2 32 38 PM](https://user-images.githubusercontent.com/583202/68889802-9ed40e00-06eb-11ea-9f29-1f99fd159b1c.png)

Reply:
But now the touch icon is simply our logo. 🙃

Reply:
Well I don't agree but I'll let it happen 😂